### PR TITLE
Pipeline, Egor Dzyumenko - 21FPL1

### PR DESCRIPTION
**Pull request by EgDzum**

### CODE REVIEW

File: lab_6_pipeline/pipeline.py [hunk 1]
```diff
@@ -18,16 +31,33 @@ def __init__(self, path_to_raw_txt_data: Path):
...
+            raise EmptyDirectoryError
+
+        if len(list(self.path_to_raw_txt_data.glob("*.json"))) != len(list(self.path_to_raw_txt_data.glob("*.txt"))):
+            raise InconsistentDatasetError
 
     def _scan_dataset(self) -> None:
         """
         Register each dataset entry
         """
+        for file in self.path_to_raw_txt_data.glob('*'):
```
Comment (dmitry-uraev) [inline lab_6_pipeline/pipeline.py]: does '*' option allows you to get all articles correctly?

**Review by dmitry-uraev — CHANGES_REQUESTED** [review]
good start
please, keep implementing

File: lab_6_pipeline/pipeline.py (hunk 1 above)
Comment (EgDzum) [reply lab_6_pipeline/pipeline.py]: changed to glob('*.txt'). List with names of files sorted by descending order.

Comment (EgDzum) [reply lab_6_pipeline/pipeline.py]: UPD: changed completely, using ```get_article_id_from_filepath``` and ```from_raw``` functions

File: lab_6_pipeline/pos_frequency_pipeline.py [hunk 2]
```diff
@@ -2,6 +2,11 @@
 Implementation of POSFrequencyPipeline for score ten only.
 """
 from typing import Optional
+from lab_6_pipeline.pipeline import *
+
+
+class EmptyFileError:
+    """sfdf"""
```
Comment (dmitry-uraev) [inline lab_6_pipeline/pos_frequency_pipeline.py:9]: Is it required somewhere?

Comment (EgDzum) [reply lab_6_pipeline/pos_frequency_pipeline.py:9]: It is required to define ```EmptyFileError``` in pos_frequency_pipeline file to execute checks locally

**Review by dmitry-uraev — APPROVED** [review]
You are done for score 4 now

**Comment by dmitry-uraev** [comment]
Make sure you have green PR till assignment

**Pull request closed without merging**